server/routes/activityRouter.js

```diff
@@ -5,11 +5,25 @@ const activityRouter = express.Router();
 
 /**
  * GET: get all activities over all students
+ * DELETE: !!!DANGER!!! delete all students
```

Comment:
Is there a way to keep this from happening?

Reply:
It works like it's supposed to; this is just letting future people know that this is a serious boy